lxd/instance/drivers/driver_qemu.go

```diff
@@ -8034,7 +8045,7 @@ func (d *qemu) checkFeatures(hostArch int, qemuPath string) (map[string]any, err
 	}
 
 	if d.architectureSupportsUEFI(hostArch) {
-		qemuArgs = append(qemuArgs, "-drive", fmt.Sprintf("if=pflash,format=raw,readonly=on,file=%s", filepath.Join(d.ovmfPath(), "OVMF_CODE.fd")))
+		qemuArgs = append(qemuArgs, "-drive", fmt.Sprintf("if=pflash,format=raw,readonly=on,file=%s", filepath.Join(d.ovmfPath(), "OVMF_CODE.4MB.fd")))
```

Comment:
Should we have a constant for "4MB" so we can use it here and on line 2885 to show these concepts are related to the same "minimum firmware size"?

Reply:
we can use `ovmfGenericFirmwares[0].code` as an option

Reply:
Sounds good thanks

Reply:
Have changed.